tests/properties.json

```diff
@@ -70,7 +70,7 @@
       "heartbeat": "/",
       "notify_path": "/notify",
       "notify_port": "18666",
-      "port": "18666",
+      "port": "8666",
```

Comment:
uses public port. This way we don't need a tunnel to the internal port.
The same should be done with CB port, but I haven't changed it yet because other features still haven't been modified